src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -180,6 +180,15 @@ public void DroppedExportProcessorItems(string exportProcessorName, string expor
             }
         }
 
+        [NonEvent]
+        public void MetricReaderEvent(string message)
```

Comment:
This serves no purpose, and can be removed.

Reply:
I have removed this.